bait-files/src/main/java/com/google/sps/data/Constants.java

```diff
@@ -0,0 +1,7 @@
+package com.google.sps.data;
```

Comment:
I thought one of these files existed already? Does it not? If not, can you look through the constants being used elsewhere and consolidate them using this file?

Reply:
Talked to Iman and this is the only Java constants for right now